core/domain/exp_services.py

```diff
@@ -627,7 +627,8 @@ def populate_exp_summary_model_fields(exp_summary_model, exp_summary):
     ExplorationSummary object.
 
     Args:
-        exp_summary_model: ExpSummaryModel. The model to populate.
+        exp_summary_model: ExpSummaryModel|None. The model to populate,
+            can be None, in that case we create a new model.
```

Comment:
The model to populate. If None, we create a new model instead.

Reply:
Done.